certified-connectors/Jupyrest/apiDefinition.swagger.json

```diff
@@ -83,19 +85,86 @@
           "200": {
             "description": "Success",
             "schema": {
-                "type": "object",
-                "properties": {
-                  "input": {
-                      "type": "object"
-                  }
+              "type": "object",
+              "properties": {
+                "input": {
+                  "type": "object"
                 }
+              }
             }
           }
         },
         "x-ms-visibility": "internal"
       }
     },
     "/NotebookExecutions": {
+      "get": {
+        "description": "Get a notebook execution.",
+        "summary": "Get a notebook execution",
+        "tags": [
+          "JupyrestConnector"
+        ],
+        "operationId": "GetNotebookExecution",
+        "deprecated": false,
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "executionId",
+            "in": "query",
+            "type": "string",
+            "required": true,
+            "x-ms-visibility": "important"
```

Comment:
@litiany23 The 'x-ms-summary' and 'description' properties are required. please add them to all parameters in the swagger file.

Reply:
done.